pkg/util/serviceaccount_test.go

```diff
@@ -187,6 +201,63 @@ func TestEnsureServiceAccountExist(t *testing.T) {
 	}
 }
 
+func TestIsServiceAccountExist(t *testing.T) {
+	type args struct {
+		client    kubernetes.Interface
+		namespace string
+		name      string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    bool
+		wantErr bool
+	}{
+		{
+			name: "not found",
+			args: args{
+				client:    fake.NewSimpleClientset(),
+				namespace: metav1.NamespaceDefault,
+				name:      "test",
+			},
+			want:    false,
+			wantErr: false,
+		},
+		{
+			name: "error",
+			args: args{
+				client:    alwaysErrorKubeClient,
+				namespace: metav1.NamespaceDefault,
+				name:      "test",
+			},
+			want:    false,
+			wantErr: true,
+		},
+		{
+			name: "error",
```

Comment:
`"error"` is the same sub-test name as above:
```suggestion
			name: "check service account exists error ",
```